pages/index.vue

```diff
@@ -191,6 +191,10 @@ export default {
     padding-bottom: 15px;
 }
 
+.list-title {
+    color: #f3cc39;
+}
+
```

Comment:
Unfortunately, https://github.com/pycontw/pycontw-2021/pull/86 has merged been ahead of this pr and has already updated the title styling. After you rebase the latest main branch, this file doesn't need any further changes.